tests/test_cif_writer.py

```diff
@@ -35,34 +36,71 @@ def test_write_single_topology_frame():
 
         # tbd replace with atom map mixin
         writer.write_frame(good_coords * 10)
+        writer.close()
+        cif = PDBxFile(temp.name)
+        assert cif.getNumFrames() == 1
+        assert cif.getPositions(asNumpy=True).shape == good_coords.shape
 
 
+@pytest.mark.parametrize("n_frames", [1, 4, 5])
 @pytest.mark.nogpu
-def test_cif_writer():
+def test_cif_writer_context(n_frames):
+    mol_a, mol_b, core = get_hif2a_ligand_pair_single_topology()
+
+    # test vacuum
+    with NamedTemporaryFile(suffix=".cif") as temp:
+        good_coords = np.concatenate([get_romol_conf(mol_a), get_romol_conf(mol_b)], axis=0)
+        with CIFWriter([mol_a, mol_b], temp.name) as writer:
+            for _ in range(n_frames):
+                writer.write_frame(good_coords * 10)
+        cif = PDBxFile(temp.name)
+        assert cif.getNumFrames() == n_frames
+        assert cif.getPositions(asNumpy=True).shape == good_coords.shape
```

Comment:
can we sanity check that coordinates are close as well? we probably can't do bitwise checks but an approximate check to .01 Angstroms is probably good

Reply:
Added in https://github.com/proteneer/timemachine/pull/1030/commits/68959d9b05f5d2fe704f9e37ee90aa58bc9fd8e8